tests/libvmops/run.go

```diff
@@ -39,65 +40,57 @@ import (
 	"kubevirt.io/client-go/kubecli"
 )
 
-func RunVMIAndExpectLaunch(vmi *v1.VirtualMachineInstance, timeout int) *v1.VirtualMachineInstance {
-	vmi, err := kubevirt.Client().VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Create(context.Background(), vmi, metav1.CreateOptions{})
-	Expect(err).ToNot(HaveOccurred())
-	By("Waiting until the VirtualMachineInstance will start")
-	return libwait.WaitForVMIPhase(vmi,
-		[]v1.VirtualMachineInstancePhase{v1.Running},
-		libwait.WithTimeout(timeout),
-	)
-}
+func RunVMAndExpectLaunchWithRunStrategy(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine, runStrategy v1.VirtualMachineRunStrategy) *v1.VirtualMachine {
```

Comment:
This method is a bit odd.
Unlike the others, it assumes the VM is already created.

I think we need to fix that.
If not in this PR, then as a follow up.

Reply:
I agree, it's only used in a couple of tests so I'll refactor it.

Reply:
Fully removed the function.